<launch>

  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
    <param name="output_frame" value="odom"/>
    <param name="freq" value="10.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="odom_used" value="false"/>
    <param name="imu_used" value="true"/>
    <param name="vo_used" value="true"/>
    <param name="debug" value="false"/>
    <param name="self_diagnose" value="false"/>
  </node>

 <node pkg="fast_klt_vo" type="fast_klt_main" name="vo" output="screen">

    <param name="start_x" value="-27.46"/>
    <param name="start_y" value ="46.6"/>
    <param name="start_z" value= "14.7"/>

    <param name="start_rot_x" value="-0.027858"/>
    <param name="start_rot_y" value="0.032963"/>
    <param name="start_rot_z" value="-0.976291"/>
    <param name="start_rot_w" value = "0.212118"/>

    <param name="min_feature_recalculate" value = "14"/>
    <param name="min_feature" value="6"/>
    <param name="ransac_prob" value="0.999"/>
    <param name="ransac_threshold" value="1.0"/>

    <param name="cov_x" value="4.0"/>
    <param name="cov_y" value="4.0"/>
    <param name="cov_z" value="2.5"/>
    <param name="cov_rot_x" value="0.3"/>
    <param name="cov_rot_y" value="0.3"/>
    <param name="cov_rot_z" value="0.3"/>
    <param name="cov_imu_x" value="0.5"/>
    <param name="cov_imu_y" value="0.5"/>
    <param name="cov_imu_z" value="0.5"/>

  </node>

</launch>
